### DISCUSSION TITLE
The Build WSA workflow is not available anymore?

### DISCUSSION THREAD

**Discussion by ghost** [LSPosed/MagiskOnWSALocal#181] [Q&A] [answered]
I was following this guide:
https://www.androidsage.com/2022/06/28/download-wsa-android-12-1-with-magisk-root-google-play-store/

I forked this repository, went to actions tab but there is no Build WSA workflow.

**Comment by Howard20181** [comment, marked as answer]
We don't build WSA using Actions, do it yourself.

**Reply by ghost** [reply]
You mean there was a "Build WSA" workflow before but you removed it? because that guide shows it was there.

**Reply by Howard20181** [reply]
Yes.

**Comment by elschopi** [comment]
Hi,
seeing this marked as answered, but to me it still leaves a few questions. I asked myself the same question as OP, since actions are referenced in the main readme text too. I fully understand the reason for not providing an action to build on the main branch here, but I'd love to have the ability to do so in my forked & detached repo.
Unfortunately, I'm not fluent enough in building with CLI, and I don't have a *nix system to use as build system. 
If anyone would be so kind and include a "How to build using actions", or something like that, it would be greatly appreciated.

**Reply by Howard20181** [reply]
Using Actions to build without detaching will cause us to be disabled, and everyone's detaching after forking will put pressure on Github's support (just now they sent an email suggesting that we should not suggest that we detach after forking but [manually clone to the local and then push it as an mirror](https://docs.github.com/en/repositories/creating-and-managing-repositories/duplicating-a-repository)). So we will not provide any support for Actions building.

**Reply by elschopi** [reply]
Okay, good to get that cleared up. I could understand the suggestion to detach. I'd have thought detaching is done through an automated system without human interaction.
Unfortunately, not being able to build with an action (as I'm used to :P ) means I can't use this any more. 
Thanks for your efforts, though :)

**Reply by Howard20181** [reply]
Before, I thought the detaching  was automatic. I didn't know it was manual until GitHub Support sending emails.

**Reply by Howard20181** [reply]
The email wrote:

> In a perfect world the Virtual Assistant would handle all of these requests autonomously and we'd have no idea how popular your repository had become! 😄 Unfortunately the nature of fork networks means we need a human to oversee the detach process, just in case something goes wrong, so the Virtual Assistant is just making sure that that human has everything they need to process the request when the ticket is created.

It seems that the detaching  process is automated, but manual observation is required to ensure that there is no problem.
